README.md

```diff
@@ -0,0 +1,7 @@
+# hrm_system
```

Comment:
MB should write a more meaningful description? What the service is and what purpose it will be used for, describe its dependencies (e.g. start DB or broker (now you can just add such heading not to forget to write about it)) and the instruction how to run services

Reply:
You are right, readme file must provide meaningful information, reworked it